service-workers/service-worker/navigationpreload.https.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>ServiceWorker: navigator.serviceWorker.navigationPreload</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/test-helpers.sub.js"></script>
+<script>
+promise_test(async t => {
+    const SCRIPT = 'resources/empty-worker.js';
+    const SCOPE = 'resources/navigationpreload';
+    let registration =
+        await service_worker_unregister_and_register(t, SCRIPT, SCOPE);
+    let np = await registration.navigationPreload;
```

Comment:
I think registration.navigationPreload is not a promise

Reply:
@mattto Many thanks for your review, I submit a new commit to address your comments, please take a look.

Reply:
I'm a little confused by github's review tool, but I think you haven't yet fixed the `await` here. It's not needed since navigationPreload isn't a promise. In fact I'd just remove `np` and test registration.navigationPreload directly (I tend to avoid abbreviations).